tuxemon/clock.py

```diff
@@ -337,11 +333,11 @@ def get_idle_time(self) -> Optional[float]:
 
         try:
             next_ts = self._scheduled_items[0].next_ts
-            return max(next_ts - self._time(), 0.0)
+            return max(next_ts - self._time, 0.0)
```

Comment:
This was a function because it needs to return the current time. You can't just call it once and then reuse it. If you want to typehint it you need to use typing.Callable